tests/stress/stress_test.go

```diff
@@ -217,175 +76,57 @@ func TestRestartNode(t *testing.T) {
-// and that the nodes remain synced
-func submitExtrinsicAssertInclusion(t *testing.T, nodes []*utils.Node, ext extrinsic.Extrinsic) {
-	tx, err := ext.Encode()
+func TestSync_Basic(t *testing.T) {
+	nodes, err := utils.InitializeAndStartNodes(t, numNodes, utils.GenesisDefault, utils.ConfigDefault)
 	require.NoError(t, err)
 
-	txStr := hex.EncodeToString(tx)
-	log.Info("submitting transaction", "tx", txStr)
-
-	// send extrinsic to random node
-	idx := rand.Intn(len(nodes))
-	prevHeader := utils.GetChainHead(t, nodes[idx]) // get starting header so that we can lookup blocks by number later
-	respBody, err := utils.PostRPC(utils.AuthorSubmitExtrinsic, utils.NewEndpoint(nodes[idx].RPCPort), "\"0x"+txStr+"\"")
+	tempDir, err := ioutil.TempDir("", "gossamer-stress-db")
 	require.NoError(t, err)
 
-	var hash modules.ExtrinsicHashResponse
-	err = utils.DecodeRPC(t, respBody, &hash)
-	require.Nil(t, err)
-	log.Info("submitted transaction", "hash", hash, "node", nodes[idx].Key)
-	t.Logf("submitted transaction to node %s", nodes[idx].Key)
-
-	// wait for nodes to build block + sync, then get headers
-	time.Sleep(time.Second * 10)
-
-	for i := 0; i < maxRetries; i++ {
-		exts := getPendingExtrinsics(t, nodes[idx])
-		if len(exts) == 0 {
-			break
-		}
-
-		time.Sleep(time.Second)
-	}
-
-	header := utils.GetChainHead(t, nodes[idx])
-	log.Info("got header from node", "header", header, "hash", header.Hash(), "node", nodes[idx].Key)
-
-	// search from child -> parent blocks for extrinsic
-	var resExts []types.Extrinsic
-	i := 0
-	for header.ExtrinsicsRoot == trie.EmptyHash && i != maxRetries {
-		// check all nodes, since it might have been included on any of the block producers
-		var block *types.Block
-
-		for j := 0; j < len(nodes); j++ {
-			block = utils.GetBlock(t, nodes[j], header.ParentHash)
-			if block == nil {
-				// couldn't get block, increment retry counter
-				i++
-				continue
-			}
-
-			header = block.Header
-			log.Info("got block from node", "hash", header.Hash(), "node", nodes[j].Key)
-			log.Debug("got block from node", "header", header, "body", block.Body, "hash", header.Hash(), "node", nodes[j].Key)
-
-			if block.Body != nil && !bytes.Equal(*(block.Body), []byte{0}) {
-				resExts, err = block.Body.AsExtrinsics()
-				require.NoError(t, err, block.Body)
-				break
-			}
-
-			if header.Hash() == prevHeader.Hash() && j == len(nodes)-1 {
-				t.Fatal("could not find extrinsic in any blocks")
-			}
-		}
-
-		if block != nil && block.Body != nil && !bytes.Equal(*(block.Body), []byte{0}) {
-			break
-		}
-	}
-
-	// assert that the extrinsic included is the one we submitted
-	require.Equal(t, 1, len(resExts), "did not find extrinsic in block on any node")
-	require.Equal(t, resExts[0], types.Extrinsic(tx))
-}
-
-func TestStress_IncludeData(t *testing.T) {
-	t.Skip()
-
-	nodes, err := utils.InitializeAndStartNodes(t, numNodes, utils.GenesisDefault)
+	db, err := scribble.New(tempDir, nil)
```

Comment:
Are we still using this db, or is it leftover from older code?  I see writes to db, but I don't see where it's read.

Reply:
we aren't using it at this point, I think the idea was to eventually create some sort of visualization. I'll remove it for now since it's unused